src/main.cpp

```diff
@@ -2898,7 +2899,8 @@ static bool ActivateBestChainStep(CValidationState &state,
                 static std::atomic<int64_t> nLastUpdate = {GetTime()};
                 if (nLastUpdate.load() < GetTime() - 5)
                 {
-                    uiInterface.NotifyBlockTip(true, pindexNewTip);
+                    uiInterface.NotifyBlockTip(IsInitialBlockDownload(), pindexNewTip);
+                    pindexLastNotify = pindexNewTip;
```

Comment:
I guess that we need this to avoid a double notification for the same block. Just wanted to understand what are the conditions to trigger the notification twice for the same height. re-org? after a period of the node being off line?